e2e/cypress/tests/support/ldap_server_commands.ts

```diff
@@ -58,7 +74,8 @@ Cypress.Commands.add('ldapModify', (filePath) => {
     ).then(({code, stdout, stderr}) => {
         cy.log(`ldapmodify code: ${code}, stdout: ${stdout}, stderr: ${stderr}`);
     });
-});
+}
+Cypress.Commands.add('ldapModify', ldapModify);
```

Comment:
we dont need this change

Reply:
do you mean `Cypress.Commands.add('ldapModify', ldapModify);` only or from `function ldapModify` to `Cypress.Commands.add('ldapModify', ldapModify);`?

Reply:
reverted

Reply:
reverted